src/Stratis.Bitcoin.Features.RPC/Controllers/FullNodeController.cs

```diff
@@ -23,8 +20,26 @@ public class FullNodeController : FeatureController
         /// <summary>Instance logger.</summary>
         private readonly ILogger logger;
 
+        private readonly IPooledTransaction pooledTransaction;
+
+        /// <summary>An interface implementation used to retrieve unspent transactions from a pooled source.</summary>
+        private readonly IPooledGetUnspentTransaction pooledGetUnspentTransaction;
+
+        /// <summary>An interface implementation used to retrieve unspent transactions.</summary>
+        private readonly IGetUnspentTransaction getUnspentTransaction;
+
+        private readonly INetworkDifficulty networkDifficulty;
+
+        /// <summary>Manager of the longest fully validated chain of blocks.</summary>
+        private readonly IConsensusLoop consensusLoop;
+
         public FullNodeController(
             ILoggerFactory loggerFactory,
+            IPooledTransaction pooledTransaction,
+            IPooledGetUnspentTransaction pooledGetUnspentTransaction,
+            IGetUnspentTransaction getUnspentTransaction,
+            INetworkDifficulty networkDifficulty,
+            IConsensusLoop consensusLoop,
```

Comment:
What is this is null? `IConsensusLoop `

Reply:
As far as I know we don't use RPC in conscienceless environments so it shouldn't be null. 
But I can add a default = null just in case if you want.

Reply:
Fixed.

Though if we are to run RPC in consensussless environments this whole class should be edited. Most of the methods cannot exist without a consensus dependency.